client/src/components/routes/event/EventDetail.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import EventDetailContainer from "containers/event/EventDetailContainer";
+
+const EventDetail = (props) => (
+  <main>
+    <EventDetailContainer eventId={props.match.params.id}/>
```

Comment:
You can place your`<EventCardContainer>` directly here, so no more need for `EventDetailContainer`, `EventDetailSection`, or `EventHorizontalScroller`.

https://github.com/compsat/ls-ganap/blob/8ff395866795bbdbe6ff31ac1b11554f45139e3c/client/src/components/routes/event/EventDetailSection.js#L22-L26

Reply:
If I get rid of the container and section like here:
<img width="575" alt="Screen Shot 2019-03-18 at 12 22 43 AM" src="https://user-images.githubusercontent.com/35568696/54494334-072bdf00-4914-11e9-9ed5-cd0281cfd846.png">

I get the same error as a while ago eh:
<img width="1054" alt="Screen Shot 2019-03-18 at 12 22 52 AM" src="https://user-images.githubusercontent.com/35568696/54494345-1d399f80-4914-11e9-9264-cee568173041.png">

Reply:
Does that error only appear if you load an event detail page directly? How about if you load the homepage first, _then_ click on an event to go to its specific page? I think I might know why. 🤔

Reply:
Yep, if I open it on the same tab after clicking "Read More", it works. Otherwise (if I open it in another tab or I go to the page directly), it returns the error

Reply:
OK, can you do the above for now and create another issue regarding this? It's a relatively trivial fix.

Reply:
In that case, can you proceed with the above suggestion and create a separate issue for this? It's a relatively trivial fix. Thanks!